app/assets/stylesheets/common/_markus.scss

```diff
@@ -358,19 +349,13 @@ fieldset {
 /** Sortable lists */
 
 .sortable {
-  @include border-radius($radius);
+  border-radius: $radius;
   background: $white;
   border: 1px solid $grey;
-
-  // Prevent text selection
-  -webkit-user-select: none;
-    apple-user-select: none;
-     -moz-user-select: none;
-      -ms-user-select: none;
-          user-select: none;
+  user-select: none;
 
   header {
-    @include border-radius-top($radius);
+    border-radius: $radius $radius 0 0;

```

Comment:
Properties should be ordered border-bottom, border-radius, font-weight, padding, text-align, text-transform